tests/integration/tier1/destructive/firewalld-config-ol8/test_firewalld_config_ol8.py

```diff
@@ -0,0 +1,40 @@
+import os
+
+import pytest
+
+from envparse import env
+
+from convert2rhel.actions.system_checks.check_firewalld_availability import FIREWALLD_CONFIG_FILE
+
+
+@pytest.mark.test_firewalld_config
+def test_firewalld_config_ol8(shell, convert2rhel):
+    """
+    Verify that properly changing the firewalld configuration file
+    allows to convert the system without any issue.
+    The reference ticket: https://issues.redhat.com/browse/RHELC-1180
+    """
+    if not os.path.exists(FIREWALLD_CONFIG_FILE):
+        pytest.fail("Firewalld configuration file does not exist")
```

Comment:
```suggestion
    if not os.path.exists(FIREWALLD_CONFIG_FILE):
        pytest.fail("Firewalld configuration file does not exist")
```
This might come and bite us in the future.
Consider this being not so probable edge case, but if for whatever reason a new image will be provided without the config, the test will fail at this point providing no valid results.
Firewalld will fall back to a default option in case no config file is provided, that being `CleanupModulesOnExit=yes`.